test/borrow-oracle/FuseOracle.ts

```diff
@@ -0,0 +1,223 @@
+import { Contract, ContractFactory } from "@ethersproject/contracts";
+import { SignerWithAddress } from "@nomiclabs/hardhat-ethers/signers";
+import hre from "hardhat";
+import { network } from "hardhat";
+import { ensureOnlyExpectedMutativeFunctions } from "../helpers";
+import { assert, addSnapshotBeforeRestoreAfterEach } from "../common";
+import {
+  currentTime,
+  toUnit,
+  fastForward,
+  assertBNGreaterThan,
+  assertBNLessThan,
+} from "../utils";
+import { expect } from "chai";
+
+const { ethers } = hre;
+const { provider, BigNumber } = ethers;
+
+const { formatEther } = require("@ethersproject/units");
+
+// Asset symbol => [gauge, underlying/eth oracle]
+// ex: rBTC-THETA => [rBTC-THETA-gauge address, BTC/ETH oracle]
+const vaults: any = {
```

Comment:
I think you can remove `: any` and typescript will implicitly type this object.

Reply:
no it gives errors otherwise